Cookbook/README.md

```diff
@@ -41,6 +41,7 @@ iOS Cookbook 👩‍🍳
     * [BBAddAdditionalPatientInformationViewControllerV2](./Technical-Documents/BBAddAdditionalPatientInformationViewControllerV2.md)
 
 * ✏️ What is Gherkin (the `Given` / `When` / `Then` / `And` formatting used by QA) and how can I determine its correctness in QA PRs
```

Comment:
I think you can remove it since in your doc there's a short description of Gherkin and `## Naming Convention` describes how to "determine correctness in PRs"
```suggestion
```